oeps/oep-38-Data-Modeling.rst

```diff
@@ -0,0 +1,438 @@
+.. vim: syntax=rst
+
+===============================
+
+OEP-38:
+
+===============================
+
++---------------+------------------------------------------------------------+
+
+| OEP | :doc:` |
+
++---------------+------------------------------------------------------------+
+
+| Title | Data Modeling Best Practices |
+
++---------------+------------------------------------------------------------+
+
+| Last Modified | 2019-06-12 |
+
++---------------+------------------------------------------------------------+
+
+| Author | Brian Beggs <bbeggs@edx.org> |
+
++---------------+------------------------------------------------------------+
+
+| Arbiter | Albemarle |
+
++---------------+------------------------------------------------------------+
+
+| Status | Provisional |
+
++---------------+------------------------------------------------------------+
+
+| Type | Architecture |
+
++---------------+------------------------------------------------------------+
+
+| Created | 2018-04-19 |
+
++---------------+------------------------------------------------------------+
```

Comment:
@macdiesel Thanks for authoring and sending this out.  Please specify a beginning and end date for the review period, as soon as possible.  Also, you or @albemarle will want to publish the review of this OEP to the appropriate #open-edx slack channels (as documented in OEP-1).  Thanks.